CMakeLists.txt

```diff
@@ -6,7 +6,10 @@
 
 cmake_minimum_required(VERSION 3.7.2)
 
-include(settings.cmake)
+if(NOT SEL4TEST_DONT_USE_SETTINGS_CMAKE)
```

Comment:
The other option I was considering is having this

```
# By default, settings.cmake is used. This can be customized:
# - use custom settings:      cmake ... -D SEL4TEST_SETTINGS_CMAKE=my-settings.cmake
# - use no settings at all:   cmake ... -D SEL4TEST_SETTINGS_CMAKE=""
if(NOT DEFINED SEL4TEST_SETTINGS_CMAKE)
    include("settings.cmake")
elseif(SEL4TEST_SETTINGS_CMAKE)
    include("${SEL4TEST_SETTINGS_CMAKE}")
endif()
```

But it turned out I never used a custom settings file anywhere, instead all custom settings were either passed on the cmake command line of by a cmake wrapper. Thus I have some doubt this is really useful.

Reply:
Do you want this PR to be merged or not? I'm confused now.

Reply:
Currently I have no immediate need for that. For the build system we had at Hensoldt Cyber, this patch used to be necessary. Not sure about the need nowadays.